Sources/Host/FreeBSD/X86_64/PTraceX86_64.cpp

```diff
@@ -9,8 +9,8 @@
 // PATENTS file in the same directory.
```

Comment:
You shouldn't touch any Linux or FreeBSD PTrace file in this commit. I think you're using an incorrect version of clang-format. You want version 3.7 or 3.8 (whichever one we're using on Travis) IIRC.